libfreerdp/codec/progressive.c

```diff
@@ -2810,6 +2815,9 @@ PROGRESSIVE_CONTEXT* progressive_context_new_ex(BOOL Compressor, UINT32 Threadin
 	progressive->rfx_context = rfx_context_new_ex(Compressor, ThreadingFlags);
 	if (!progressive->rfx_context)
 		goto fail;
+	if (!progressive->rfx_context->dwt_2d_extrapolate_decode)
```

Comment:
wrong place, move to where `dwt_2d_decode` is initialized to default (e.g. `rfx_context_new_ex`)